corehq/apps/accounting/tests/test_credit_lines.py

```diff
@@ -308,7 +309,7 @@ def test_transfers(self):
             edition=SoftwarePlanEdition.STANDARD
         )
         first_sub = Subscription.new_domain_subscription(
-            self.account, self.domain, advanced_plan
+            self.account, self.domain.name, advanced_plan
```

Comment:
Adding ```deactivate_subscriptions``` (https://github.com/dimagi/commcare-hq/pull/12463/files#diff-a297c8348184ff3201dbf1b13fb0921eR340) affects this test for reasons in https://github.com/dimagi/commcare-hq/pull/12465

The code change also affects the workflow in this test, because we should wait until a subscription is deactivated to transfer credits instead of transferring them when a future subscription is created.